packages/client-react-streaming/src/DataTransportAbstraction/WrappedApolloClient.tsx

```diff
@@ -175,29 +113,24 @@ export class ApolloClientBrowserImpl<
 
     const canonicalVariables = canonicalStringify(options.variables || {});
 
-    const cacheKey = [print(serverQuery), canonicalVariables].toString();
+    const cacheKeyArr = [print(serverQuery), canonicalVariables];
+    const cacheKey = JSON.stringify(cacheKeyArr);
 
-    return { query: serverQuery, cacheKey, varJson: canonicalVariables };
+    return {
+      cacheKey,
+      cacheKeyArr,
+    };
   }
 
-  onQueryStarted = ({
-    options,
-    id,
-  }: Extract<QueryEvent, { type: "started" }>) => {
-    const hydratedOptions = {
-      ...options,
-      query: gql(options.query),
-    };
-    const { query, varJson, cacheKey } =
-      this.identifyUniqueQuery(hydratedOptions);
+  onQueryStarted({ options, id }: Extract<QueryEvent, { type: "started" }>) {
```

Comment:
This had to become a bound method so I could overwrite it in a child and still call `super.onQueryStarted`